.travis.yml

```diff
@@ -1,6 +1,5 @@
 language: node_js
 node_js:
-  - "7"
-  - "6"
-  - "4"
+  - "8"
+  - "10"
```

Comment:
By the end of today Node 12 will be LTS, worth adding now? Agree with adding even numbers only

Reply:
I am waiting for a confirmed release number to give it a test on my local before raising a PR.

Reply:
Added node v12 in travis config